src/test/java/openfasttrack/importer/specobject/TestSpecobjectImporter.java

```diff
@@ -84,31 +62,27 @@ public void setup()
     @Test
     public void testSingleSpecObject() throws FileNotFoundException
     {
-        final Map<SpecificationItemId, SpecificationItem> result = runImporter(
-                "single-specobject.xml");
-        assertThat(result.values(), hasSize(1));
-        assertThat(result, hasEntry(this.specItem1.getId(), this.specItem1));
+        final List<SpecificationItem> result = runImporter("single-specobject.xml");
```

Comment:
Now it is an integration test (accesses resource). I think we should follow the Maven naming rules for ITests.